samples/highcharts/demo/bar-force/demo.js

```diff
@@ -0,0 +1,178 @@
+                                y + topMargin + (cutRight ? len : 0)
+                            ], // arrow
+                            // top side
+                            ['L', x + w, y + topMargin],
+                            // right side
+                            ['L', x + w, y + h],
+                            // bottom side
+                            [
+                                'L',
+                                x + w / 2,
+                                y + h + bottomMargin + (cutLeft ? len : 0)
+                            ],
+                            ['L', x, y + h + bottomMargin],
+                            // left side
+                            ['L', x, y],
+                            // close
+                            ['Z']
+                        ]
+                    };
+
+                });
+            }
+        }
+    );
+}(Highcharts));
+
+Highcharts.chart('container', {
+    chart: {
+        type: 'bar',
+        height: 170,
+        events: {
+            render: function () {
+                const chart = this,
+                    renderer = chart.renderer,
+                    top = chart.plotTop,
+                    left = chart.plotLeft,
+                    yAxis = chart.yAxis[0];
+
+                if (!this.minDataLabel) {
+                    this.minDataLabel = renderer.label(
+                        'Min: ' + yAxis.dataMin
+                    ).attr({ zIndex: 4 }).css({ fontWeight: 'bold' }).add();
+                }
+                if (!this.maxDataLabel) {
+                    this.maxDataLabel = renderer.label(
+                        'Max: ' + yAxis.dataMax
+                    ).attr({ zIndex: 4 }).css({ fontWeight: 'bold' }).add();
+                }
+
+                this.minDataLabel.attr({
+                    x: left +
+                        yAxis.toPixels(yAxis.dataMin, true) -
+                        (this.minDataLabel.width / 2),
+                    y: top
+                });
+                this.maxDataLabel.attr({
+                    x: left +
+                        yAxis.toPixels(yAxis.dataMax, true) -
+                        (this.maxDataLabel.width / 2),
+                    y: top
+                });
+            }
+        }
+    },
+    title: {
+        text: 'Estimated Impact of Forces in Mars EDL sequence'
+    },
+    plotOptions: {
+        bar: {
+            headSize: 6,
+            stacking: 'normal',
+            dataLabels: {
+                enabled: true,
+                y: 20,
+                verticalAlign: 'bottom'
+            },
+            color: 'rgb(255, 7, 77)',
+            negativeColor: 'rgb(1, 127, 250)',
+            accessibility: {
+                exposeAsGroupOnly: true
+            }
+        }
+    },
+    tooltip: {
+        format: '<span style="color:{point.color}">\u25CF</span> ' +
+            '<b>{series.name}: {point.y}</b>',
+        stickOnContact: true
```

Comment:
I think we should remove this. The tooltip gets in the way of hovering to read the values.